Started GET "/" for 192.168.0.1 at 2021-02-17 00:45:08 +0000
  
ActiveRecord::ConnectionNotEstablished (FATAL:  role "root" does not exist
):
  
activerecord (6.1.2.1) lib/active_record/connection_adapters/postgresql_adapter.rb:83:in `rescue in new_client'
activerecord (6.1.2.1) lib/active_record/connection_adapters/postgresql_adapter.rb:77:in `new_client'
activerecord (6.1.2.1) lib/active_record/connection_adapters/postgresql_adapter.rb:37:in `postgresql_connection'
activerecord (6.1.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:882:in `public_send'
activerecord (6.1.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:882:in `new_connection'
activerecord (6.1.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:926:in `checkout_new_connection'
activerecord (6.1.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:905:in `try_to_checkout_new_connection'
activerecord (6.1.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:866:in `acquire_connection'
activerecord (6.1.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:588:in `checkout'
activerecord (6.1.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:428:in `connection'
activerecord (6.1.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:1128:in `retrieve_connection'
activerecord (6.1.2.1) lib/active_record/connection_handling.rb:327:in `retrieve_connection'
activerecord (6.1.2.1) lib/active_record/connection_handling.rb:283:in `connection'
activerecord (6.1.2.1) lib/active_record/migration.rb:611:in `connection'
activerecord (6.1.2.1) lib/active_record/migration.rb:606:in `build_watcher'
activerecord (6.1.2.1) lib/active_record/migration.rb:588:in `block in call'
activerecord (6.1.2.1) lib/active_record/migration.rb:587:in `synchronize'
activerecord (6.1.2.1) lib/active_record/migration.rb:587:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
activesupport (6.1.2.1) lib/active_support/callbacks.rb:98:in `run_callbacks'
actionpack (6.1.2.1) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/actionable_exceptions.rb:18:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/debug_exceptions.rb:29:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.1.2.1) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.1.2.1) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.1.2.1) lib/active_support/tagged_logging.rb:99:in `block in tagged'
activesupport (6.1.2.1) lib/active_support/tagged_logging.rb:37:in `tagged'
activesupport (6.1.2.1) lib/active_support/tagged_logging.rb:99:in `tagged'
railties (6.1.2.1) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/request_id.rb:26:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.1.2.1) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/static.rb:24:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/host_authorization.rb:98:in `call'
railties (6.1.2.1) lib/rails/engine.rb:539:in `call'
puma (5.2.1) lib/puma/configuration.rb:247:in `call'
puma (5.2.1) lib/puma/request.rb:76:in `block in handle_request'
puma (5.2.1) lib/puma/thread_pool.rb:337:in `with_force_shutdown'
puma (5.2.1) lib/puma/request.rb:75:in `handle_request'
puma (5.2.1) lib/puma/server.rb:431:in `process_client'
puma (5.2.1) lib/puma/thread_pool.rb:145:in `block in spawn_thread'
Started GET "/favicon.ico" for 192.168.0.1 at 2021-02-17 00:45:09 +0000
  
ActiveRecord::ConnectionNotEstablished (FATAL:  role "root" does not exist
):
  
activerecord (6.1.2.1) lib/active_record/connection_adapters/postgresql_adapter.rb:83:in `rescue in new_client'
activerecord (6.1.2.1) lib/active_record/connection_adapters/postgresql_adapter.rb:77:in `new_client'
activerecord (6.1.2.1) lib/active_record/connection_adapters/postgresql_adapter.rb:37:in `postgresql_connection'
activerecord (6.1.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:882:in `public_send'
activerecord (6.1.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:882:in `new_connection'
activerecord (6.1.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:926:in `checkout_new_connection'
activerecord (6.1.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:905:in `try_to_checkout_new_connection'
activerecord (6.1.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:866:in `acquire_connection'
activerecord (6.1.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:588:in `checkout'
activerecord (6.1.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:428:in `connection'
activerecord (6.1.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:1128:in `retrieve_connection'
activerecord (6.1.2.1) lib/active_record/connection_handling.rb:327:in `retrieve_connection'
activerecord (6.1.2.1) lib/active_record/connection_handling.rb:283:in `connection'
activerecord (6.1.2.1) lib/active_record/migration.rb:611:in `connection'
activerecord (6.1.2.1) lib/active_record/migration.rb:606:in `build_watcher'
activerecord (6.1.2.1) lib/active_record/migration.rb:588:in `block in call'
activerecord (6.1.2.1) lib/active_record/migration.rb:587:in `synchronize'
activerecord (6.1.2.1) lib/active_record/migration.rb:587:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
activesupport (6.1.2.1) lib/active_support/callbacks.rb:98:in `run_callbacks'
actionpack (6.1.2.1) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/actionable_exceptions.rb:18:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/debug_exceptions.rb:29:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.1.2.1) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.1.2.1) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.1.2.1) lib/active_support/tagged_logging.rb:99:in `block in tagged'
activesupport (6.1.2.1) lib/active_support/tagged_logging.rb:37:in `tagged'
activesupport (6.1.2.1) lib/active_support/tagged_logging.rb:99:in `tagged'
railties (6.1.2.1) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/request_id.rb:26:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.1.2.1) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/static.rb:24:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/host_authorization.rb:98:in `call'
railties (6.1.2.1) lib/rails/engine.rb:539:in `call'
puma (5.2.1) lib/puma/configuration.rb:247:in `call'
puma (5.2.1) lib/puma/request.rb:76:in `block in handle_request'
puma (5.2.1) lib/puma/thread_pool.rb:337:in `with_force_shutdown'
puma (5.2.1) lib/puma/request.rb:75:in `handle_request'
puma (5.2.1) lib/puma/server.rb:431:in `process_client'
puma (5.2.1) lib/puma/thread_pool.rb:145:in `block in spawn_thread'
Started GET "/" for 192.168.0.1 at 2021-02-17 00:46:10 +0000
  
ActiveRecord::NoDatabaseError (FATAL:  database "petstore_backend_development" does not exist
):
  
activerecord (6.1.2.1) lib/active_record/connection_adapters/postgresql_adapter.rb:81:in `rescue in new_client'
activerecord (6.1.2.1) lib/active_record/connection_adapters/postgresql_adapter.rb:77:in `new_client'
activerecord (6.1.2.1) lib/active_record/connection_adapters/postgresql_adapter.rb:37:in `postgresql_connection'
activerecord (6.1.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:882:in `public_send'
activerecord (6.1.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:882:in `new_connection'
activerecord (6.1.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:926:in `checkout_new_connection'
activerecord (6.1.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:905:in `try_to_checkout_new_connection'
activerecord (6.1.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:866:in `acquire_connection'
activerecord (6.1.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:588:in `checkout'
activerecord (6.1.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:428:in `connection'
activerecord (6.1.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:1128:in `retrieve_connection'
activerecord (6.1.2.1) lib/active_record/connection_handling.rb:327:in `retrieve_connection'
activerecord (6.1.2.1) lib/active_record/connection_handling.rb:283:in `connection'
activerecord (6.1.2.1) lib/active_record/migration.rb:611:in `connection'
activerecord (6.1.2.1) lib/active_record/migration.rb:606:in `build_watcher'
activerecord (6.1.2.1) lib/active_record/migration.rb:588:in `block in call'
activerecord (6.1.2.1) lib/active_record/migration.rb:587:in `synchronize'
activerecord (6.1.2.1) lib/active_record/migration.rb:587:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
activesupport (6.1.2.1) lib/active_support/callbacks.rb:98:in `run_callbacks'
actionpack (6.1.2.1) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/actionable_exceptions.rb:18:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/debug_exceptions.rb:29:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.1.2.1) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.1.2.1) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.1.2.1) lib/active_support/tagged_logging.rb:99:in `block in tagged'
activesupport (6.1.2.1) lib/active_support/tagged_logging.rb:37:in `tagged'
activesupport (6.1.2.1) lib/active_support/tagged_logging.rb:99:in `tagged'
railties (6.1.2.1) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/request_id.rb:26:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.1.2.1) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/static.rb:24:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/host_authorization.rb:98:in `call'
railties (6.1.2.1) lib/rails/engine.rb:539:in `call'
puma (5.2.1) lib/puma/configuration.rb:247:in `call'
puma (5.2.1) lib/puma/request.rb:76:in `block in handle_request'
puma (5.2.1) lib/puma/thread_pool.rb:337:in `with_force_shutdown'
puma (5.2.1) lib/puma/request.rb:75:in `handle_request'
puma (5.2.1) lib/puma/server.rb:431:in `process_client'
puma (5.2.1) lib/puma/thread_pool.rb:145:in `block in spawn_thread'
Started GET "/favicon.ico" for 192.168.0.1 at 2021-02-17 00:46:11 +0000
  
ActiveRecord::NoDatabaseError (FATAL:  database "petstore_backend_development" does not exist
):
  
activerecord (6.1.2.1) lib/active_record/connection_adapters/postgresql_adapter.rb:81:in `rescue in new_client'
activerecord (6.1.2.1) lib/active_record/connection_adapters/postgresql_adapter.rb:77:in `new_client'
activerecord (6.1.2.1) lib/active_record/connection_adapters/postgresql_adapter.rb:37:in `postgresql_connection'
activerecord (6.1.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:882:in `public_send'
activerecord (6.1.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:882:in `new_connection'
activerecord (6.1.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:926:in `checkout_new_connection'
activerecord (6.1.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:905:in `try_to_checkout_new_connection'
activerecord (6.1.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:866:in `acquire_connection'
activerecord (6.1.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:588:in `checkout'
activerecord (6.1.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:428:in `connection'
activerecord (6.1.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:1128:in `retrieve_connection'
activerecord (6.1.2.1) lib/active_record/connection_handling.rb:327:in `retrieve_connection'
activerecord (6.1.2.1) lib/active_record/connection_handling.rb:283:in `connection'
activerecord (6.1.2.1) lib/active_record/migration.rb:611:in `connection'
activerecord (6.1.2.1) lib/active_record/migration.rb:606:in `build_watcher'
activerecord (6.1.2.1) lib/active_record/migration.rb:588:in `block in call'
activerecord (6.1.2.1) lib/active_record/migration.rb:587:in `synchronize'
activerecord (6.1.2.1) lib/active_record/migration.rb:587:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
activesupport (6.1.2.1) lib/active_support/callbacks.rb:98:in `run_callbacks'
actionpack (6.1.2.1) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/actionable_exceptions.rb:18:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/debug_exceptions.rb:29:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.1.2.1) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.1.2.1) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.1.2.1) lib/active_support/tagged_logging.rb:99:in `block in tagged'
activesupport (6.1.2.1) lib/active_support/tagged_logging.rb:37:in `tagged'
activesupport (6.1.2.1) lib/active_support/tagged_logging.rb:99:in `tagged'
railties (6.1.2.1) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/request_id.rb:26:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.1.2.1) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/static.rb:24:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/host_authorization.rb:98:in `call'
railties (6.1.2.1) lib/rails/engine.rb:539:in `call'
puma (5.2.1) lib/puma/configuration.rb:247:in `call'
puma (5.2.1) lib/puma/request.rb:76:in `block in handle_request'
puma (5.2.1) lib/puma/thread_pool.rb:337:in `with_force_shutdown'
puma (5.2.1) lib/puma/request.rb:75:in `handle_request'
puma (5.2.1) lib/puma/server.rb:431:in `process_client'
puma (5.2.1) lib/puma/thread_pool.rb:145:in `block in spawn_thread'
  [1m[35m (127.2ms)[0m  [1m[35mCREATE DATABASE "petstore_backend_development" ENCODING = 'unicode'[0m
  [1m[35m (103.3ms)[0m  [1m[35mCREATE DATABASE "petstore_backend_test" ENCODING = 'unicode'[0m
Started GET "/" for 192.168.16.1 at 2021-02-17 00:51:18 +0000
Processing by Rails::WelcomeController#index as HTML
  Rendering /usr/local/bundle/gems/railties-6.1.2.1/lib/rails/templates/rails/welcome/index.html.erb
  Rendered /usr/local/bundle/gems/railties-6.1.2.1/lib/rails/templates/rails/welcome/index.html.erb (Duration: 15.6ms | Allocations: 290)
Completed 200 OK in 61ms (Views: 41.4ms | ActiveRecord: 0.0ms | Allocations: 1557)


Started GET "/favicon.ico" for 192.168.16.1 at 2021-02-17 00:51:19 +0000
  
ActionController::RoutingError (No route matches [GET] "/favicon.ico"):
  
Started GET "/" for 192.168.16.1 at 2021-02-17 00:58:42 +0000
Processing by Rails::WelcomeController#index as HTML
  Rendering /usr/local/bundle/gems/railties-6.1.2.1/lib/rails/templates/rails/welcome/index.html.erb
  Rendered /usr/local/bundle/gems/railties-6.1.2.1/lib/rails/templates/rails/welcome/index.html.erb (Duration: 25.5ms | Allocations: 290)
Completed 200 OK in 65ms (Views: 48.2ms | ActiveRecord: 0.0ms | Allocations: 1557)


Started GET "/" for 192.168.16.1 at 2021-02-17 00:59:04 +0000
Processing by Rails::WelcomeController#index as HTML
  Rendering /usr/local/bundle/gems/railties-6.1.2.1/lib/rails/templates/rails/welcome/index.html.erb
  Rendered /usr/local/bundle/gems/railties-6.1.2.1/lib/rails/templates/rails/welcome/index.html.erb (Duration: 6.5ms | Allocations: 178)
Completed 200 OK in 10ms (Views: 9.3ms | ActiveRecord: 0.0ms | Allocations: 718)


Started GET "/" for 192.168.16.1 at 2021-02-17 01:03:36 +0000
Processing by Rails::WelcomeController#index as HTML
  Rendering /usr/local/bundle/gems/railties-6.1.2.1/lib/rails/templates/rails/welcome/index.html.erb
  Rendered /usr/local/bundle/gems/railties-6.1.2.1/lib/rails/templates/rails/welcome/index.html.erb (Duration: 9.9ms | Allocations: 293)
Completed 200 OK in 42ms (Views: 23.7ms | ActiveRecord: 0.0ms | Allocations: 2269)


  [1m[35m (66.7ms)[0m  [1m[35mCREATE TABLE "schema_migrations" ("version" character varying NOT NULL PRIMARY KEY)[0m
  [1m[35m (6.3ms)[0m  [1m[35mCREATE TABLE "ar_internal_metadata" ("key" character varying NOT NULL PRIMARY KEY, "value" character varying, "created_at" timestamp(6) NOT NULL, "updated_at" timestamp(6) NOT NULL)[0m
  [1m[35m (1.5ms)[0m  [1m[34mSELECT pg_try_advisory_lock(293181872227134030)[0m
  [1m[35m (1.5ms)[0m  [1m[34mSELECT "schema_migrations"."version" FROM "schema_migrations" ORDER BY "schema_migrations"."version" ASC[0m
Migrating to CreateUsers (20210217190826)
  [1m[36mTRANSACTION (0.6ms)[0m  [1m[35mBEGIN[0m
  [1m[35m (21.5ms)[0m  [1m[35mCREATE TABLE "users" ("id" bigserial primary key, "email" character varying, "password_digest" character varying, "created_at" timestamp(6) NOT NULL, "updated_at" timestamp(6) NOT NULL)[0m
  [1m[36mActiveRecord::SchemaMigration Create (3.9ms)[0m  [1m[32mINSERT INTO "schema_migrations" ("version") VALUES ($1) RETURNING "version"[0m  [["version", "20210217190826"]]
  [1m[36mTRANSACTION (2.3ms)[0m  [1m[35mCOMMIT[0m
  [1m[36mActiveRecord::InternalMetadata Load (0.7ms)[0m  [1m[34mSELECT "ar_internal_metadata".* FROM "ar_internal_metadata" WHERE "ar_internal_metadata"."key" = $1 LIMIT $2[0m  [["key", "environment"], ["LIMIT", 1]]
  [1m[36mTRANSACTION (0.4ms)[0m  [1m[35mBEGIN[0m
  [1m[36mActiveRecord::InternalMetadata Create (3.2ms)[0m  [1m[32mINSERT INTO "ar_internal_metadata" ("key", "value", "created_at", "updated_at") VALUES ($1, $2, $3, $4) RETURNING "key"[0m  [["key", "environment"], ["value", "development"], ["created_at", "2021-02-17 19:16:17.223187"], ["updated_at", "2021-02-17 19:16:17.223187"]]
  [1m[36mTRANSACTION (1.4ms)[0m  [1m[35mCOMMIT[0m
  [1m[35m (0.6ms)[0m  [1m[34mSELECT pg_advisory_unlock(293181872227134030)[0m
  [1m[35m (2.2ms)[0m  [1m[34mSELECT "schema_migrations"."version" FROM "schema_migrations" ORDER BY "schema_migrations"."version" ASC[0m
  [1m[36mUser Load (1.4ms)[0m  [1m[34mSELECT "users".* FROM "users" ORDER BY "users"."id" ASC LIMIT $1[0m  [["LIMIT", 1]]
  [1m[36mTRANSACTION (0.3ms)[0m  [1m[35mBEGIN[0m
  [1m[36mUser Create (2.1ms)[0m  [1m[32mINSERT INTO "users" ("email", "password_digest", "created_at", "updated_at") VALUES ($1, $2, $3, $4) RETURNING "id"[0m  [["email", "saad.18@gmail.com"], ["password_digest", "$2a$12$tA1pBY1OynZDx4lwyC./ke2O8KvBeJgTAbwggnmox7jyvar/A86s6"], ["created_at", "2021-02-17 20:32:41.540432"], ["updated_at", "2021-02-17 20:32:41.540432"]]
  [1m[36mTRANSACTION (0.7ms)[0m  [1m[35mCOMMIT[0m
  [1m[35m (5.4ms)[0m  [1m[34mSELECT COUNT(*) FROM "users"[0m
  [1m[36mUser Load (1.7ms)[0m  [1m[34mSELECT "users".* FROM "users" ORDER BY "users"."id" ASC LIMIT $1[0m  [["LIMIT", 1]]
  [1m[36mUser Load (0.6ms)[0m  [1m[34mSELECT "users".* FROM "users" WHERE "users"."email" = $1 LIMIT $2[0m  [["email", "saad.18@gmail.com"], ["LIMIT", 1]]
  [1m[36mUser Load (0.6ms)[0m  [1m[34mSELECT "users".* FROM "users" WHERE "users"."email" = $1 LIMIT $2[0m  [["email", "saad.18@gmail.com"], ["LIMIT", 1]]
  [1m[36mUser Load (0.8ms)[0m  [1m[34mSELECT "users".* FROM "users" ORDER BY "users"."id" ASC LIMIT $1[0m  [["LIMIT", 1]]
  [1m[36mUser Load (0.6ms)[0m  [1m[34mSELECT "users".* FROM "users" WHERE "users"."email" = $1 LIMIT $2[0m  [["email", "saad.18@gmail.com"], ["LIMIT", 1]]
  [1m[36mUser Load (0.6ms)[0m  [1m[34mSELECT "users".* FROM "users" WHERE "users"."email" = $1 LIMIT $2[0m  [["email", "saad.18@gmail.com"], ["LIMIT", 1]]
  [1m[36mUser Load (0.6ms)[0m  [1m[34mSELECT "users".* FROM "users" WHERE "users"."id" = $1 LIMIT $2[0m  [["id", 1], ["LIMIT", 1]]
Started POST "/api/v1/authentication" for 192.168.16.1 at 2021-02-17 21:09:01 +0000
  [1m[35m (1.7ms)[0m  [1m[34mSELECT "schema_migrations"."version" FROM "schema_migrations" ORDER BY "schema_migrations"."version" ASC[0m
  
ActionController::RoutingError (No route matches [POST] "/api/v1/authentication"):
  
Started POST "/api/v1/authenticate" for 192.168.16.1 at 2021-02-17 21:09:09 +0000
  
ActionController::RoutingError (uninitialized constant AuthenticationController
Did you mean?  AuthenticationServices):
  
Started POST "/api/v1/authenticate" for 192.168.16.1 at 2021-02-17 21:09:28 +0000
  
ActionController::RoutingError (uninitialized constant AuthenticationController
Did you mean?  AuthenticationServices):
  
Started POST "/api/v1/authenticate" for 192.168.16.1 at 2021-02-17 21:09:50 +0000
  
ActionController::RoutingError (uninitialized constant AuthenticationController
Did you mean?  AuthenticationServices):
  
Started POST "/api/v1/authenticate" for 192.168.16.1 at 2021-02-17 21:10:59 +0000
  [1m[35m (2.8ms)[0m  [1m[34mSELECT "schema_migrations"."version" FROM "schema_migrations" ORDER BY "schema_migrations"."version" ASC[0m
  
ActionController::RoutingError (uninitialized constant AuthenticationController
Did you mean?  AuthenticationServices):
  
Started POST "/api/v1/authenticate" for 192.168.16.1 at 2021-02-17 21:15:12 +0000
  [1m[35m (1.6ms)[0m  [1m[34mSELECT "schema_migrations"."version" FROM "schema_migrations" ORDER BY "schema_migrations"."version" ASC[0m
Processing by Api::V1::AuthenticationController#authenticate as HTML
  Parameters: {"email"=>"saad.18@gmail.com", "password"=>"[FILTERED]", "authentication"=>{"email"=>"saad.18@gmail.com", "password"=>"[FILTERED]"}}
Completed 500 Internal Server Error in 5ms (ActiveRecord: 0.0ms | Allocations: 392)


  
ArgumentError (wrong number of arguments (given 1, expected 0; required keyword: headers)):
  
app/services/authentication_services/authorize_api_call_service.rb:5:in `call'
app/controllers/api/v1/base_controller.rb:8:in `authenticate_request'
Started POST "/api/v1/authenticate" for 192.168.16.1 at 2021-02-17 21:15:44 +0000
Processing by Api::V1::AuthenticationController#authenticate as HTML
  Parameters: {"email"=>"saad.18@gmail.com", "password"=>"[FILTERED]", "authentication"=>{"email"=>"saad.18@gmail.com", "password"=>"[FILTERED]"}}
  [1m[36mUser Load (1.3ms)[0m  [1m[34mSELECT "users".* FROM "users" WHERE "users"."email" = $1 LIMIT $2[0m  [["email", "saad.18@gmail.com"], ["LIMIT", 1]]
  ↳ app/services/authentication_services/authenticate_user_action.rb:9:in `block in <class:AuthenticateUserAction>'
Completed 200 OK in 349ms (Views: 0.2ms | ActiveRecord: 13.2ms | Allocations: 5428)


Started GET "/api/v1/test" for 192.168.16.1 at 2021-02-17 21:18:38 +0000
  [1m[35m (2.1ms)[0m  [1m[34mSELECT "schema_migrations"."version" FROM "schema_migrations" ORDER BY "schema_migrations"."version" ASC[0m
Processing by Api::V1::TestController#index as HTML
  Parameters: {"test"=>{}}
Completed 500 Internal Server Error in 8ms (ActiveRecord: 0.0ms | Allocations: 395)


  
ArgumentError (wrong number of arguments (given 1, expected 0; required keyword: headers)):
  
app/services/authentication_services/authorize_api_call_service.rb:5:in `call'
app/controllers/api/v1/base_controller.rb:8:in `authenticate_request'
Started GET "/api/v1/test" for 192.168.16.1 at 2021-02-17 21:18:58 +0000
Processing by Api::V1::TestController#index as HTML
  Parameters: {"test"=>{}}
  [1m[36mUser Load (0.9ms)[0m  [1m[34mSELECT "users".* FROM "users" WHERE "users"."id" = $1 LIMIT $2[0m  [["id", 1], ["LIMIT", 1]]
  ↳ app/services/authentication_services/get_user_action.rb:9:in `block in <class:GetUserAction>'
Completed 200 OK in 93ms (Views: 1.7ms | ActiveRecord: 10.6ms | Allocations: 5319)


Started GET "/api/v1/test" for 192.168.16.1 at 2021-02-17 21:19:04 +0000
Processing by Api::V1::TestController#index as HTML
  Parameters: {"test"=>{}}
Completed 500 Internal Server Error in 2ms (ActiveRecord: 0.0ms | Allocations: 172)


  
JWT::DecodeError (Not enough or too many segments):
  
app/services/json_web_token.rb:9:in `decode'
app/services/authentication_services/extract_auth_header_action.rb:11:in `block in <class:ExtractAuthHeaderAction>'
app/services/authentication_services/authorize_api_call_service.rb:6:in `call'
app/controllers/api/v1/base_controller.rb:8:in `authenticate_request'
Started GET "/api/v1/test" for 192.168.16.1 at 2021-02-17 21:19:12 +0000
Processing by Api::V1::TestController#index as HTML
  Parameters: {"test"=>{}}
Completed 500 Internal Server Error in 0ms (ActiveRecord: 0.0ms | Allocations: 165)


  
JWT::DecodeError (Not enough or too many segments):
  
app/services/json_web_token.rb:9:in `decode'
app/services/authentication_services/extract_auth_header_action.rb:11:in `block in <class:ExtractAuthHeaderAction>'
app/services/authentication_services/authorize_api_call_service.rb:6:in `call'
app/controllers/api/v1/base_controller.rb:8:in `authenticate_request'
Started GET "/api/v1/test" for 192.168.16.1 at 2021-02-17 21:19:44 +0000
Processing by Api::V1::TestController#index as HTML
  Parameters: {"test"=>{}}
Started GET "/api/v1/test" for 192.168.16.1 at 2021-02-17 21:21:37 +0000
  [1m[35m (3.2ms)[0m  [1m[34mSELECT "schema_migrations"."version" FROM "schema_migrations" ORDER BY "schema_migrations"."version" ASC[0m
Processing by Api::V1::TestController#index as HTML
  Parameters: {"test"=>{}}
Completed 500 Internal Server Error in 349425ms (ActiveRecord: 0.0ms | Allocations: 45012)


  
JWT::DecodeError (Not enough or too many segments):
  
app/services/json_web_token.rb:9:in `decode'
app/services/authentication_services/extract_auth_header_action.rb:12:in `block in <class:ExtractAuthHeaderAction>'
app/services/authentication_services/authorize_api_call_service.rb:6:in `call'
app/controllers/api/v1/base_controller.rb:8:in `authenticate_request'
Started GET "/api/v1/test" for 192.168.16.1 at 2021-02-17 21:27:29 +0000
Processing by Api::V1::TestController#index as HTML
  Parameters: {"test"=>{}}
Completed 500 Internal Server Error in 6804ms (ActiveRecord: 0.0ms | Allocations: 3749)


  
NoMethodError (undefined method `headers' for #<LightService::Context:0x00007fe93c12bd08>):
  
app/services/authentication_services/extract_auth_header_action.rb:10:in `block in <class:ExtractAuthHeaderAction>'
app/services/authentication_services/authorize_api_call_service.rb:6:in `call'
app/controllers/api/v1/base_controller.rb:8:in `authenticate_request'
Started GET "/api/v1/test" for 192.168.16.1 at 2021-02-17 21:28:06 +0000
Processing by Api::V1::TestController#index as HTML
  Parameters: {"test"=>{}}
  [1m[36mUser Load (3.3ms)[0m  [1m[34mSELECT "users".* FROM "users" WHERE "users"."id" = $1 LIMIT $2[0m  [["id", 1], ["LIMIT", 1]]
  ↳ app/services/authentication_services/get_user_action.rb:9:in `block in <class:GetUserAction>'
Completed 200 OK in 101ms (Views: 1.0ms | ActiveRecord: 27.7ms | Allocations: 5195)


Started GET "/api/v1/test" for 192.168.16.1 at 2021-02-17 21:28:15 +0000
Processing by Api::V1::TestController#index as HTML
  Parameters: {"test"=>{}}
Completed 500 Internal Server Error in 30ms (ActiveRecord: 0.0ms | Allocations: 708)


  
JWT::DecodeError (Not enough or too many segments):
  
app/services/json_web_token.rb:9:in `decode'
app/services/authentication_services/extract_auth_header_action.rb:9:in `block in <class:ExtractAuthHeaderAction>'
app/services/authentication_services/authorize_api_call_service.rb:6:in `call'
app/controllers/api/v1/base_controller.rb:8:in `authenticate_request'
Started GET "/api/v1/test" for 192.168.16.1 at 2021-02-17 21:28:23 +0000
Processing by Api::V1::TestController#index as HTML
  Parameters: {"test"=>{}}
Completed 500 Internal Server Error in 45179ms (ActiveRecord: 0.0ms | Allocations: 10623)


  
JWT::DecodeError (Not enough or too many segments):
  
app/services/json_web_token.rb:9:in `decode'
app/services/authentication_services/extract_auth_header_action.rb:10:in `block in <class:ExtractAuthHeaderAction>'
app/services/authentication_services/authorize_api_call_service.rb:6:in `call'
app/controllers/api/v1/base_controller.rb:8:in `authenticate_request'
Started GET "/api/v1/test" for 192.168.16.1 at 2021-02-17 21:29:12 +0000
Processing by Api::V1::TestController#index as HTML
  Parameters: {"test"=>{}}
Filter chain halted as :authenticate_request rendered or redirected
Completed 401 Unauthorized in 52ms (Views: 0.2ms | ActiveRecord: 0.0ms | Allocations: 785)


Started GET "/api/v1/test" for 192.168.16.1 at 2021-02-17 21:29:20 +0000
Processing by Api::V1::TestController#index as HTML
  Parameters: {"test"=>{}}
Filter chain halted as :authenticate_request rendered or redirected
Completed 401 Unauthorized in 2ms (Views: 0.2ms | ActiveRecord: 0.0ms | Allocations: 197)


Started GET "/api/v1/test" for 192.168.16.1 at 2021-02-17 21:29:24 +0000
Processing by Api::V1::TestController#index as HTML
  Parameters: {"test"=>{}}
  [1m[36mUser Load (0.4ms)[0m  [1m[34mSELECT "users".* FROM "users" WHERE "users"."id" = $1 LIMIT $2[0m  [["id", 1], ["LIMIT", 1]]
  ↳ app/services/authentication_services/get_user_action.rb:9:in `block in <class:GetUserAction>'
Completed 200 OK in 25ms (Views: 0.2ms | ActiveRecord: 5.6ms | Allocations: 3398)


Started GET "/api/v1/test" for 192.168.16.1 at 2021-02-17 21:32:31 +0000
  [1m[35m (2.7ms)[0m  [1m[34mSELECT "schema_migrations"."version" FROM "schema_migrations" ORDER BY "schema_migrations"."version" ASC[0m
Processing by Api::V1::TestController#index as HTML
  Parameters: {"test"=>{}}
  [1m[36mUser Load (1.2ms)[0m  [1m[34mSELECT "users".* FROM "users" WHERE "users"."id" = $1 LIMIT $2[0m  [["id", 1], ["LIMIT", 1]]
  ↳ app/services/authentication_services/get_user_action.rb:9:in `block in <class:GetUserAction>'
Completed 200 OK in 113ms (Views: 0.3ms | ActiveRecord: 9.6ms | Allocations: 7095)


Started POST "/api/v1/authenticate" for 192.168.16.1 at 2021-02-17 21:34:05 +0000
Processing by Api::V1::AuthenticationController#authenticate as HTML
  Parameters: {"email"=>"saad.18@gmail.com", "password"=>"[FILTERED]", "authentication"=>{"email"=>"saad.18@gmail.com", "password"=>"[FILTERED]"}}
  [1m[36mUser Load (0.6ms)[0m  [1m[34mSELECT "users".* FROM "users" WHERE "users"."email" = $1 LIMIT $2[0m  [["email", "saad.18@gmail.com"], ["LIMIT", 1]]
  ↳ app/services/authentication_services/authenticate_user_action.rb:9:in `block in <class:AuthenticateUserAction>'
Completed 200 OK in 312ms (Views: 0.2ms | ActiveRecord: 0.6ms | Allocations: 1316)


Started GET "/api/v1/test" for 192.168.16.1 at 2021-02-17 21:34:28 +0000
Processing by Api::V1::TestController#index as HTML
  Parameters: {"test"=>{}}
  [1m[36mUser Load (0.5ms)[0m  [1m[34mSELECT "users".* FROM "users" WHERE "users"."id" = $1 LIMIT $2[0m  [["id", 1], ["LIMIT", 1]]
  ↳ app/services/authentication_services/get_user_action.rb:9:in `block in <class:GetUserAction>'
Completed 200 OK in 7ms (Views: 0.3ms | ActiveRecord: 0.5ms | Allocations: 898)


Started POST "/api/v1/authenticate" for 192.168.80.1 at 2021-02-17 22:28:16 +0000
  [1m[35m (1.6ms)[0m  [1m[34mSELECT "schema_migrations"."version" FROM "schema_migrations" ORDER BY "schema_migrations"."version" ASC[0m
Processing by Api::V1::AuthenticationController#authenticate as HTML
  Parameters: {"email"=>"saad.18@gmail.com", "password"=>"[FILTERED]", "authentication"=>{"email"=>"saad.18@gmail.com", "password"=>"[FILTERED]"}}
  [1m[36mUser Load (2.0ms)[0m  [1m[34mSELECT "users".* FROM "users" WHERE "users"."email" = $1 LIMIT $2[0m  [["email", "saad.18@gmail.com"], ["LIMIT", 1]]
  ↳ app/services/authentication_services/authenticate_user_action.rb:9:in `block in <class:AuthenticateUserAction>'
Completed 201 Created in 370ms (Views: 0.4ms | ActiveRecord: 17.8ms | Allocations: 7825)


